ci_scripts/k8s-splunk.yml

```diff
@@ -1,4 +1,47 @@
 apiVersion: v1
+kind: ConfigMap
```

Comment:
Might suggest exposing k8s-splunk.yml more to users as demo in the examples read me with something like this. Not required though.

## Deploy local Splunk Enterprise Demo

Deploy the docker image for Splunk Enterprise to a cluster.

```bash
curl https://raw.githubusercontent.com/signalfx/splunk-otel-collector-chart/main/ci_scripts/k8s-splunk.yml | kubectl apply -f -
```

Use these Helm Chart values to deploy with one of the exampleslocal Splunk instance.

```yaml
# Splunk Platform required parameters
clusterName: splunk
splunkPlatform:
  token: "00000000-0000-0000-0000-0000000000000"
  index: main
```